src/Illuminate/Console/Scheduling/CommandBuilder.php

```diff
@@ -51,6 +51,10 @@ protected function buildBackgroundCommand(Event $event)
 
         $finished = Application::formatCommandString('schedule:finish').' "'.$event->mutexName().'"';
 
+        if (windows_os()) {
+            return 'start /b cmd /c "('.$event->command.' & '.$finished.')'.$redirect.$output.' 2>&1"';
+        }
+
         return $this->ensureCorrectUser($event,
             '('.$event->command.$redirect.$output.' 2>&1 '.(windows_os() ? '&' : ';').' '.$finished.') > '
```

Comment:
It seems the windows_os() check is redundant here ?

Reply:
Yes! I removed it